### PR TITLE
gclient: Add dev Vara node spawning

### PR DESCRIPTION
**Pull request by shamilsan**
- Made it possible to spawn a Vara node in the dev-mode.

@gear-tech/dev

### CODE REVIEW

File: gclient/src/node/mod.rs [hunk 1]
```diff
@@ -36,16 +36,45 @@ pub struct Node {
...
+    pub fn ws_address(&self) -> &WSAddress {
+        &self.ws_address
+    }
+
+    fn try_node_from_path(path: impl AsRef<OsStr>, args: Vec<&str>) -> Result<Self> {
         let port = port::pick();
         let port_string = port.to_string();
 
-        let args = vec!["--ws-port", &port_string, "--tmp", "--dev"];
+        let mut args = args;
```
Comment (gshep) [inline gclient/src/node/mod.rs:75]: looks like there is no need for mutable rebinding. Argument can be marked `mut` directly in paratemeters list

**Review by gshep — APPROVED** [review]
:+1:

File: gclient/src/node/mod.rs (hunk 1 above)
Comment (shamilsan) [reply gclient/src/node/mod.rs:75]: I don't like `mut` in the function signature when it is not required there by the function contract as it looks a little bit confusing in my opinion.

Comment (shamilsan) [reply gclient/src/node/mod.rs:75]: But after reading this comment I've changed my mind: https://stackoverflow.com/questions/31388644/whats-the-idiomatic-way-to-pass-by-mutable-value#comment50755158_31388745

Comment (shamilsan) [reply gclient/src/node/mod.rs:75]: But I get another error 🙈 

```
error[E0597]: `port_string` does not live long enough
  --> gclient/src/node/mod.rs:77:19
   |
71 |     fn try_node_from_path(path: impl AsRef<OsStr>, mut args: Vec<&str>) -> Result<Self> {
   |                                                                  - let's call the lifetime of this reference `'1`
...
77 |         args.push(&port_string);
   |         ----------^^^^^^^^^^^^-
   |         |         |
   |         |         borrowed value does not live long enough
   |         argument requires that `port_string` is borrowed for `'1`
...
93 |     }
   |     - `port_string` dropped here while still borrowed
```

So let's keep it as is.

**Pull request merged**